docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Transfer_your_app_from_appc_CLI_to_ti_CLI/README.md

```diff
@@ -0,0 +1,45 @@
+# Transfer your app from `appc` CLI to `ti` CLI
+
+## Introduction
+
+Axway announced that they will hand over Titanum to the community at <a href="https://devblog.axway.com/featured/product-update-changes-to-application-development-services-appcelerator/" target="_blank">March 2022</a>. <b>You still create apps or use existing apps!</b>. But you won't have access to the Axway services (e.g. Push, Analytics). You might also see a message like `You cannot perform this operation with your current plan. Please upgrade to build your application for production distribution`. It's time to update the CLI!
```

Comment:
Maybe clean up the wording. The "they will" has already happened.

Reply:
fixed